h-w2

```diff
@@ -0,0 +1,86 @@
+#1
+#1.1
+
+data = [
+    {'name': 'Viktor', 'city': 'Kiev', 'age': 30 },
+    {'name': 'Maksim', 'city': 'Dnepr', 'age': 20},
+    {'name': 'Vladimir', 'city': 'Lviv', 'age': 32},
+    {'name': 'Andrey', 'city': 'Kiev', 'age': 34},
+    {'name': 'Artem', 'city': 'Dnepr', 'age': 50},
+    {'name': 'Dmitriy', 'city': 'Lviv', 'age': 21}]
+data.sort(key=lambda d: d['age'])
+print (t)
+
+#1.2
+
+data = [
+    {'name': 'Viktor', 'city': 'Kiev', 'age': 30 },
+    {'name': 'Maksim', 'city': 'Dnepr', 'age': 20},
+    {'name': 'Vladimir', 'city': 'Lviv', 'age': 32},
+    {'name': 'Andrey', 'city': 'Kiev', 'age': 34},
+    {'name': 'Artem', 'city': 'Dnepr', 'age': 50},
+    {'name': 'Dmitriy', 'city': 'Lviv', 'age': 21}]
+    data.sort(key=lambda x:x['city'][:6])
+    data
+from itertools import groupby
+for k,v in groupby(data,key=lambda x:x['city'][:6]):
+result=(k, list(v))
```

Comment:
four-space is required inside `for` loop.
Also, the result of your solution is not quite accurate.
YOUR RESULT : 

```
('Dnepr', [{'name': 'Maksim', 'city': 'Dnepr', 'age': 20}])
('Lviv', [{'name': 'Dmitriy', 'city': 'Lviv', 'age': 21}])
('Kiev', [{'name': 'Viktor', 'city': 'Kiev', 'age': 30}])
('Lviv', [{'name': 'Vladimir', 'city': 'Lviv', 'age': 32}])
('Kiev', [{'name': 'Andrey', 'city': 'Kiev', 'age': 34}])
('Dnepr', [{'name': 'Artem', 'city': 'Dnepr', 'age': 50}])
```

REQUIRED RESULT: in description